tests/ha/check_iscsi_initiator.py

```diff
@@ -0,0 +1,91 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Verify iSCSI Initiator Name is not the same on different installations
+# Maintainer: qe-sap@suse.com
+
+from testapi import *
+from mmapi import get_current_job_id
+
+perl.require("serial_terminal")
+perl.require("power_action_utils")
+
+def run(self):
+    """
+    NAME: ha/check_iscsi_initiator.py
+
+    MAINTAINER: QE-SAP <qe-sap@suse.de>
+
+    DESCRIPTION: verify iSCSI Initiator Name is not the same after different
+                 installations.
+
+                 This module needs to be scheduled at least 2 times in the
+                 same test. On the first time it will store the iSCSI Initiator
+                 Name from the SUT in a runtime setting, and on the second it
+                 will compare the iSCSI Initiator Name from the SUT against the
+                 stored in the setting. It will fail is both Initiator Names are
+                 the same.
+
+                 It is expected that this module is scheduled after different
+                 installations. If scheduled twice after the same installation,
+                 it will fail.
+
+                 iSCSI Initiator Name is assumed to be stored in
+                 /etc/iscsi/initiatorname.iscsi. If the file does not exist, assume
+                 no Initiator Name is configured by default and exit successfully.
+
+    OPENQA SETTINGS:
+
+    * INST_AUTO: setting providing agama with an unattended JSONnet file.
+
+    * ISCSI_INITIATOR_{jobid}: runtime setting where the initiator from the first
+                               installation is stored.
+    """
+    perl.serial_terminal.select_serial_terminal()
+    iscsi_initiator_file = "/etc/iscsi/initiatorname.iscsi"
+
+    # Verify if a default iSCSI initiator is set on the SUT
+    if (not script_run(f"test -f {iscsi_initiator_file}")):
+        record_info("No iSCSI Initiator", f"SUT has no {iscsi_initiator_file}");
```

Comment:
I think there should be no semicolon at the end right?

Reply:
Oh. right! :D